glidertest/tools.py

```diff
@@ -1089,6 +1157,30 @@ def plot_vertical_speeds_with_histograms(ds, start_prof=None, end_prof=None):
 
     ax2_hist.legend(loc=legend_loc)
 
+    # Set font sizes for all annotations
+    # Font size 14 looks roughly like fontsize 8 when I drop this figure in Word - a bit small

```

Comment:
The same here. try looping over these axes with

`for axes in [ax1, ax2, ax1_hist, ax2_hist]`

Reply:
Done.  Should I be marking these as 'resolve conversation'

Reply:
I think we can trust a professor to mark their own home work! Feel free to mark as resolved if it's a simple fix like this. Maybe leave it for the reviewer to do if it's more complex/subjective